full/Dockerfile

```diff
@@ -8,6 +8,15 @@ RUN yes | unminimize \
         build-essential \
         htop \
         jq \
+        hub=2.7.0~ds1-1 \
+        zsh=5.5.1-1ubuntu3 \
+        tmux=2.8-3 \
+        fzf=0.17.5-2 \
+        mtr=0.92-2 \
+        ripgrep=0.10.0-2 \
```

Comment:
We have grep

Reply:
See #76

Reply:
We already use `rg` in Gitpod (it's the back-end for the workspace search -- that's why it's so mind-blowingly fast).

But it would be cool to have `rg` accessible from the Terminal as well. Maybe we should pay attention not to install it twice (unless its installed size is negligible).